### PR TITLE
Non USCM Personnel, headbands, tshirts and dcc uniform

### LINKED ISSUE #187: Survivor sprites

### LINKED ISSUE #189: Survivor ftl

### LINKED ISSUE #190: Survivor spawnpoint(s)

### LINKED ISSUE #191: Combat Correspondent sprites

### LINKED ISSUE #193: Combat Correspondent ftl

### LINKED ISSUE #194: Combat Correspondent spawnpoint

### LINKED ISSUE #514: Press Body Armor
https://cm-ss13.com/wiki/Marine_Equipment#Press_Body_Armor

### LINKED ISSUE #516: Press Helmet
https://cm-ss13.com/wiki/Marine_Equipment#Press_Helmet

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR
Adds Combat Correspondent, Survivor, clothes for them, Press armor, headbands, tshirts and also DCC uniform cuz i forgor.

## Why / Balance
Resolves #187, resolves #189, resolves #190, resolves #191, resolves #193, resolves #194, resolves #514, resolves #516

## Media

image_2023-11-19_16-09-39

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Tunguso4ka
- add: Combat Correspondent and Survivors has been added to the game.

### CODE REVIEW

**Comment by Tunguso4ka** [comment]
fuck you "ComputeManagedAssemblies" task was not found.

**Comment by Tunguso4ka** [comment]
Ready to review.

File: Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml:1]: Probably just use the empty "hudsquad" for these?

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- type: statusIcon
+  parent: CMJobIconBase
+  id: JobIconCorrespondent
+  icon:
+    sprite: _CM14/Interface/cm_job_icons.rsi
+    state: hudsquad_ass #idk whose this icon
+
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml:7]: .

File: Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- type: statusIcon
+  parent: CMJobIconBase
+  id: JobIconSurvivor
+  icon:
+    sprite: _CM14/Interface/cm_job_icons.rsi
+    state: hudsquad_ass #todo cm14 survivor icon
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml:13]: .

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/_CM14/department.ftl [hunk 3]
```diff
@@ -25,3 +25,7 @@ department-MarineMedical-description = Treat all patients that come into medbay.
 # auxiliary support
 department-AuxiliarySupport = Auxiliary Support
 department-AuxiliarySupport-description = The Auxiliary support personnel are tasked with supporting the marines either by providing support through firepower or other means.
+
+# non uscm
+department-Civilians = Non USCM Personnel
```
[the remark this answers is not in the document]
Comment (Tunguso4ka) [reply Resources/Locale/en-US/_CM14/department.ftl]: done.

File: Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml (hunk 1 above)
Comment (Tunguso4ka) [reply Resources/Prototypes/_CM14/Roles/Jobs/JobIcons/other_job_icons.yml:7]: done.

File: Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    role: JobHospitalCorpsman
+    time: 18000 #5 hours
+  - !type:RoleTimeRequirement
+    role: JobCombatTech
+    time: 18000 #5 hours
+  startingGear: SurvivorGear
+  icon: "JobIconSurvivor"
+  requireAdminNotify: false
+  joinNotifyCrew: false
+  supervisors: job-supervisors-command # no one probably?
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml:21]: Wiki says they respond to nobody but themselves
They have no idea who the marines are or their intentions

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml (hunk 4 above)
Comment (Tunguso4ka) [reply Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml]: done.

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**